metaphlan/utils/read_fastx.py

```diff
@@ -114,10 +114,15 @@ def read_and_write_raw_int(fd, min_len=None, prefix_id=""):
     #         avg_read_length = len(l) + avg_read_length
     #         _ = sys.stdout.write(ignore_spaces(l))
 
+    if not idx:
+        sys.stderr.write('Error: no reads found.\n')
+        sys.exit(1)
+
     nreads = idx - discarded
 
     if not nreads:
-        nreads, avg_read_length = 0, 0
+        sys.stderr.write('Error: no reads longer than {} bp found.\n'.format(min_len))
+        sys.exit(1)
```

Comment:
This will fail anyway because it will divide by zero on original line 172. This was the error I had to troubleshoot.

Also, while I didn't make this change because it's out of scope for this PR, the variable here shouldn't be called `avg_read_length`. Te average read length is actually calculated later by dividing `avg_read_length` (which is actually the total read length and should be called `total_read_length` or something of that nature) by `nreads`.